src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/DebtorsController.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.companieshouse.web.accounts.controller.smallfull;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import uk.gov.companieshouse.web.accounts.annotation.NextController;
+import uk.gov.companieshouse.web.accounts.annotation.PreviousController;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+import uk.gov.companieshouse.web.accounts.controller.ConditionalController;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.debtors.Debtors;
+import uk.gov.companieshouse.web.accounts.service.smallfull.BalanceSheetService;
+import uk.gov.companieshouse.web.accounts.service.smallfull.DebtorsService;
+import uk.gov.companieshouse.web.accounts.validation.ValidationError;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
+@NextController(ReviewController.class)
+@PreviousController(OtherAccountingPolicyController.class)
+@RequestMapping("/company/{companyNumber}/transaction/{transactionId}/company-accounts" +
+        "/{companyAccountsId}/small-full/debtors")
+public class DebtorsController extends BaseController implements ConditionalController {
+
+    @Autowired
+    private DebtorsService debtorsService;
+
+    @Autowired
+    private BalanceSheetService balanceSheetService;
+
+    @Override
+    protected String getTemplateName() {
+        return "smallfull/debtors";
+    }
+
+    @GetMapping
+    public String getDebtors(@PathVariable String companyNumber,
+            @PathVariable String transactionId,
+            @PathVariable String companyAccountsId,
+            Model model, HttpServletRequest request) throws ServiceException {
```

Comment:
Remove the throws you are catching it and never re-throwing it.

Reply:
Removed [here](https://github.com/companieshouse/company-accounts.web.ch.gov.uk/commit/af432e5dfd7b0c6a59c3e49a3f4374b1c8885ac8) within the [bug fix PR](https://github.com/companieshouse/company-accounts.web.ch.gov.uk/pull/159) which will be merged into this branch.